packages/rules-unit-testing/src/api/index.ts

```diff
@@ -157,6 +158,10 @@ export type FirebaseEmulatorOptions = {
   };
 };
 
+function trimmedBase64Encode(val: string): string {
+  return base64Encode(val).split("=")[0];
```

Comment:
Our utility library uses `.` as the padding character for URL-safe alphabet, causing the error you saw. Suggested fix:

```suggestion
  // Use base64url encoding and remove padding in the end (dot characters).
  return base64Encode(val).replace(/\./g,"");
```